tests/Dfc.CourseDirectory.WebV2.Tests/LayoutTests.cs

```diff
@@ -345,6 +345,56 @@ public async Task ProviderUserForBothFEAndApprenticeships_RendersExpectedNav(Tes
             Assert.Equal(expectedHref, bulkUploadLink.GetAttribute("href"));
         }
 
+        [Theory]
+        [InlineData(TestUserType.ProviderSuperUser)]
+        [InlineData(TestUserType.ProviderUser)]
+        public async Task ProviderUserProviderNotPassedQA_DoesNotRenderApprenticeshipsLink(TestUserType userType)
+        {
+            // Arrange
+            var providerId = await TestData.CreateProvider(
+                providerType: ProviderType.Both,
+                providerName: "Test Provider",
+                apprenticeshipQAStatus: ApprenticeshipQAStatus.NotStarted);
+
+            await User.AsTestUser(userType, providerId);
+
+            // Act
+            var response = await HttpClient.GetAsync($"/tests/empty-provider-context");
+
+            // Assert
+            response.EnsureSuccessStatusCode();
+
+            var doc = await response.GetDocument();
+            var topLevelLinks = GetTopLevelNavLinks(doc);
+            var topLevelLinkLabels = topLevelLinks.Select(l => l.Label);
+            Assert.DoesNotContain("Your apprenticeships training", topLevelLinkLabels);
```

Comment:
This could easily become a false-positive it the link text is changed.
You could use `data-qa` attributes to make it less fragile https://dev.to/chriszie/data-qa-attribute-a-better-way-to-select-elements-for-ui-test-automation-48lm